i18n/en.json

```
@@ -263,6 +263,14 @@
     "id": "api.channel.delete_channel.type.invalid",
     "translation": "Unable to delete direct or group message channels"
   },
+  {
+    "id": "api.channel.undelete_channel.unarchived",
+    "translation": "%v unarchived the channel."
+  },
+  {
+    "id": "api.channel.undelete_channel.undeleted.app_error",
+    "translation": "The channel has been unarchived"
```

That seams like a misleading error message. What about
```suggestion
    "translation": "Unable to unarchived a not archived channel"
```